docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -5,66 +5,83 @@
-   ```sh
-   sudo -E ./install_NEO_OCL_driver.sh
-   ```
-   > **NOTE**: To use the **Intel® Iris® Xe MAX Graphics**, see the [Intel® Iris® Xe MAX Graphics with Linux*](https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html) page for driver installation instructions.
+
+   2. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. The drivers are not included in the package. To install, run this script:
+
+   .. code-block:: sh
+
+      sudo -E ./install_NEO_OCL_driver.sh
+
+   .. note:: To use the **Intel® Iris® Xe MAX Graphics**, see the `Intel® Iris® Xe MAX Graphics with Linux`_ page for driver installation instructions.
+
+      .. _Intel® Iris® Xe MAX Graphics with Linux: https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html
    
-   The script compares the driver version on the system to the current version. If the driver version on the system is higher or equal to the current version, the script does 
-   not install a new driver. If the version of the driver is lower than the current version, the script uninstalls the lower version and installs the current version with your permission:
-   ![](../img/NEO_check_agreement.png) 
-
-   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output:  
-   ```sh
-   Add OpenCL user to video group    
-   ```
+   The script compares the driver version on the system to the current version. If the driver version on the system is higher or equal to the current version, the script does not install a new driver. If the version of the driver is lower than the current version, the script uninstalls the lower version and installs the current version with your permission:
+
+   .. image:: ../img/NEO_check_agreement.png
+
+   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output: 
+
+   .. code-block:: sh
+
+      Add OpenCL user to video group    
+
    Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
 
-3. **Optional:** Install header files to allow compilation of new code. You can find the header files at [Khronos OpenCL™ API Headers](https://github.com/KhronosGroup/OpenCL-Headers.git).
+   You may also find the most recent version of the driver, installation procedure and other information on the `Intel® software for general purpose GPU capabilities`_ site.
```

Comment:
```suggestion
   The most recent version of the driver, installation procedure and other information can also be find on the `Intel® software for general purpose GPU capabilities`_ site.
```

Reply:
> 

```suggestion
 The most recent version of the driver, installation procedure and other information can also be found on the `Intel® software for general purpose GPU capabilities`_ site.
```

Passive voice takes the past participle form of a verb.